Source/WebKit/Shared/WTFArgumentCoders.serialization.in

```diff
@@ -49,6 +49,11 @@ header: <wtf/text/AtomString.h>
     String string()
 }
 
+header: <wtf/CheckedRef.h>
+[CustomHeader, Nested] class WTF::CanMakeCheckedPtr {
```

Comment:
Interesting.  Not technically nested, but this does the trick.  I'm not sure how I feel about this, but we can always make it marginally nicer later if we feel like it can be improved.